.projenrc.ts

```diff
@@ -50,7 +50,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   // dependency config
   jsiiVersion: '~5.0.0',
   cdkVersion: '2.110.0',
-  bundledDeps: ['esbuild'] /* Runtime dependencies of this module. */,
+  bundledDeps: [] /* Runtime dependencies of this module. */,
```

Comment:
cool if this works...

Reply:
I don't think we need esbuild bundled because we don't import {} from "esbuild" anywhere. We also don't use `NodejsFunction` anywhere now (because open-next bundles for us) which is what depends upon esbuild

Reply:
sweet